### PR TITLE
Test/186502/unit tests

### PR DESCRIPTION
**Pull request by spicy-rice**
- Recreated react app using vite
- inc Typescript 
- Jest unit tests

### CODE REVIEW

**Comment by jrabbott** [comment]
Please could we review and clean-up the folder structure with the front-end-components dir.

We seem to have front-end & frontend, several src, etc.

**Review by joekadi — CHANGES_REQUESTED** [review]
folder reorg

**Review by joekadi — DISMISSED** [review]

**Review by jrabbott — CHANGES_REQUESTED** [review]
Please could we review and clean-up the folder structure with the front-end-components dir.

We seem to have front-end & frontend, several src, etc.

File: front-end-components/src/App.test.tsx
Comment (jrabbott) [inline front-end-components/src/App.test.tsx:1]: With *.test,tsx do we have any preferences to whether they are placed with the *.tsx file in the src directory? or whether we place them in the tests directory?

Comment (jrabbott) [reply front-end-components/src/App.test.tsx:1]: Lets stick with that for now then. We can always change it in future if it comes challenging.

**Review by jrabbott — APPROVED** [review]

**Pull request merged**